activities/models/emoji.py

```diff
@@ -258,6 +259,7 @@ def by_ap_tag(cls, domain: Domain, data: dict, create: bool = False):
 
     ### Mastodon API ###
 
+    @cache_memoize(120)
```

Comment:
`cache_memoize` makes this method slower. We've already paid the cost of fetching the instance and `Config.system` is loaded at the start of requests and periodically by Stator.